torch_geometric/loader/neighbor_loader.py

```diff
@@ -159,8 +165,10 @@ def _sparse_neighbor_sample(self, index: Tensor):
         )
         return node, row, col, edge
 
-    def _hetero_sparse_neighbor_sample(self, index_dict: Dict[str, Tensor],
+    def _hetero_sparse_neighbor_sample(self, index_dict: Dict[str,
+                                                              torch.Tensor],
                                        **kwargs):
```

Comment:
```suggestion
                                       **kwargs,):
```
+ let yapf auto-format :)